src/tabs/borderless.scss

```diff
@@ -28,6 +28,30 @@
     }
   }
 
+  // #region Horizontal tabs
```

Comment:
Is it worth having such a small regions?

Reply:
I'll remove them in the next push.